### PR TITLE
cert-manager v0.8 migration guide update

### PR DESCRIPTION
**Pull request by nefischer**

### CODE REVIEW

File: docs/how-to-docs/cert-manager-upgrade-from-v0.8.md [hunk 1]
```diff
@@ -59,25 +59,32 @@ The main draw-back of this approach is that the value for the new secret being c
...
-  - For `Ingress` resources, add the new `cert-manager.io` annotations and labels
-  - For `Certificate` resources that are part of your deployments (e.g. to create a certificate that is mounted by an nginx sidecar for your main service), amend the `Certificate`'s annotations and labels
-- Deploy the new changes. You should now have a new set of certificate resources. You can verify this with `kubectl -n project get certificate.cert-manager.io`. Note that will list the new `Certificate` resources that you created as well as the ones automatically created by cert-manager on your behalf to manage ingress certificates.
-- Identify all the secrets associated with the `Certificate` resources listed in the previous step (secrets have the same name as their associated `Certificate` resources)
-  - Back them up
-  - Delete them with `kubectl -n project delete secret xxx`. This is required because the current secrets still have annotations related to the old version of cert-manager
-  - Watch as the secrets get re-created by the new version of cert-manager
+The high levels steps are:
+
+- Leave the current `Ingress` resource as it is (whith old v0.8 annoations)
```
Comment (KashifSaadat) [inline docs/how-to-docs/cert-manager-upgrade-from-v0.8.md:72]: `annoations` => `annotations`

File: docs/how-to-docs/cert-manager-upgrade-from-v0.8.md [hunk 2]
```diff
@@ -59,25 +59,32 @@ The main draw-back of this approach is that the value for the new secret being c
...
+- Create a `Certificate` resource with `letsencrypt-prod` as the clusterIssuer, a new secret name and the appropriate stanzas as shown later on this guide
+- Deploy the changes to create the new certificate resource. Please note that the certificate and associated secret will at that point be unused.
+- Update your `Ingress` resources
+  - Remove all `certmanager.k8s.io` annotations
+  - DO NOT add any new `cert-manager.io` annotations or labels
+  - Update `secretName` in the `Ingress` resource to the name of the secret you created in step 2
+- Deploy the `Ingress` changes
+- When you've checked that the service is functioning as intended, you can tidy up the old v0.8 cert-manager resources:
+  - delete any certificate resources still returned by `kubectl -n project get certificate` (back them up if they are not stored in git)
```
Comment (KashifSaadat) [inline docs/how-to-docs/cert-manager-upgrade-from-v0.8.md:81]: To be safe, be more explicit and say `certificates.certmanager.k8s.io`?

Comment (nefischer) [reply docs/how-to-docs/cert-manager-upgrade-from-v0.8.md]: good point; updated

File: docs/how-to-docs/cert-manager-upgrade-from-v0.8.md (hunk 1 above)
Comment (nefischer) [reply docs/how-to-docs/cert-manager-upgrade-from-v0.8.md]: updated

**Pull request merged**